utilities/dot-project/yaml/example-project.yaml

```diff
@@ -0,0 +1,66 @@
+name: "Kubernetes"
+description: "Kubernetes is an open-source system for automating deployment, scaling, and management of containerized applications."
+schema_version: "1.0.0"
+type: "platform"
+maturity_log:
+  - phase: "sandbox"
+    date: 2016-03-10T00:00:00Z
+    issue: "https://github.com/cncf/toc/issues/123"
+  - phase: "incubating" 
+    date: 2017-03-01T00:00:00Z
+    issue: "https://github.com/cncf/toc/issues/123"
+  - phase: "graduated"
+    date: 2018-03-06T00:00:00Z
+    issue: "https://github.com/cncf/toc/issues/123"
+repositories:
+  - "https://github.com/kubernetes/kubernetes"
+  - "https://github.com/kubernetes/kubectl"
+  - "https://github.com/kubernetes/client-go"
+  - "https://github.com/kubernetes/api"
+social:
+  twitter: "kubernetesio"
+  slack: "kubernetes.slack.com"
+  youtube: "KubernetesCommunity" 
+  linkedin: "kubernetes"
+  reddit: "r/kubernetes"
+  stackoverflow: "kubernetes"
```

Comment:
I know they can put whatever here, but we'll probably want to mirror the landscape and have them provide urls

e.g. slack_url